build/resources/main/META-INF/persistence.xml

```diff
@@ -1 +1,24 @@
-
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"

```

Comment:
The `build` dir should not be in source (it is in the `.gitignore`). Do a `git rm -r --cached build/` to fix this.